napari/layers/surface/surface.py

```diff
@@ -441,7 +441,7 @@ def _extent_data(self) -> np.ndarray:
         return extrema
 
     @property
-    def features(self):
+    def features(self) -> pd.DataFrame:
```

Comment:
Given the docstring, is there anything that would make mypy happy while preserving the idea that this doesn't return a pandas DF?

Reply:
(and also slightly further down)

Reply:
It does (at the moment) always return a DataFrame, so I think this is the correct annotation (at least for now).